apps/etterlatte-behandling/src/test/kotlin/grunnlagsendring/doedshendelse/DoedshendelseDaoTest.kt

```diff
@@ -38,4 +38,10 @@ class DoedshendelseDaoTest(val dataSource: DataSource) {
         doedshendelseDao.hentDoedshendelserMedStatus(DoedshendelseStatus.NY) shouldBe listOf(doedshendelse)
         doedshendelseDao.hentDoedshendelserMedStatus(DoedshendelseStatus.OPPDATERT) shouldBe emptyList()
     }
+
+    @Test
+    fun `Skal oppdatere hendelse ferdig med brevid`() {
+        // TODO: må først ha egen versjon grunnlagsendringshendelseService.opprettHendelseAvTypeForPerson som setter sak_id
```

Comment:
Denne logikken regner jeg med ikke havner i `grunnlagsendringshendelseService.opprettHendelseAvTypeForPerson`? Men kan ta det når vi kommer dit. Kanskje like greit å bare fjerne testen for nå?

Reply:
Tenkte det var fint med en reminder egentlig :P